src/docs/Components/OptionGroup/OptionCard/previews/extended.preview.html

```diff
@@ -0,0 +1,14 @@
+<div class="example">
```

Comment:
To be more consistent, I think you should rename the file to `default.html.twig` or `radio.html.twig` (same for the SCSS file)